generatebundlefile/hack/release.sh

```diff
@@ -30,7 +30,7 @@ EOF
 # Release Package Images to Packages Artifact account
 BASE_DIRECTORY=$(git rev-parse --show-toplevel)
 export AWS_CONFIG_FILE=${BASE_DIRECTORY}/generatebundlefile/configfile
```

Comment:
I don't think that export is going to do anything, but I could be missing something

Reply:
It is a supported field for the CLI https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html